pkg/apis/projectcalico/v3/packetcapture.go

```diff
@@ -60,6 +60,23 @@ type PacketCaptureSpec struct {
 	// The ordered set of filters applied to traffic captured from an interface.  Each rule contains a set of
 	// packet match criteria.
 	Filters []PacketCaptureRule `json:"filters,omitempty" validate:"omitempty,dive"`
+
+	// Coordinates the scheduled start/stop capturing traffic for a PacketCapture
+	// +optional
+	AutoTermination *PacketCaptureAutoTermination `json:"autoTermination,omitempty" validate:"omitempty"`
```

Comment:
I think I'd prefer to name this positively: more about what it does than what it doesn't do.

I'm not sure I have a great name though - I think there's a question in there about whether we should have separate sections for time and for file size related questions.
- If we want to keep it very general, maybe something like CaptureConditions or RunConditions?
- If we want to separate time and size then we could bring the Start/EndTime up to the top level (I think they wouldn't need any extra name words), or perhaps have a Schedule block or some such.

I think I'm leaning slightly towards having the times just in the top level, and using a separate block for size things.

Reply:
I agree with the name suggestion. `RunConditions` sounds better.